snooty/main.py

```diff
@@ -1,13 +1,14 @@
 """Snooty.
 
 Usage:
-  snooty build <source-path> [<mongodb-url>] [--commit=<commit_hash>]
+  snooty build <source-path> [<mongodb-url>] [(--commit=<commit_hash> |(--commit=<commit_hash> --patch=<patch_id>))]
```

Comment:
nit but 
* Should there be a space after `|`?
* Do we need that outer `(...)` expression? e.g. can it just be `[--commit=<commit_hash> | (--commit=<commit_hash> --patch=<patch_id>)]`